config/routes.rb

```diff
@@ -77,6 +77,8 @@
   resources :categories, :except => [:show, :update, :destroy, :edit]
   resources :categories, :path => 'category', :only => [:show, :edit, :update, :destroy]
   match '/category/:id/page/:page', :to => 'categories#show', :format => false
+  match '/categories/page/:page', :to => 'categories#index', :format => false
```

Comment:
I'm not sure you need to add a route - i think this should work without a new route